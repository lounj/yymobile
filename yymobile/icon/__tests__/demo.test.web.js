import { webDemoTest } from '../../../tests/shared/yyDemoTest';

webDemoTest('icon');
