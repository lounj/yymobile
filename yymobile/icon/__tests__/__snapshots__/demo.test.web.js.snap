// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./yymobile/icon/demo/size.md correctly 1`] = `
<div
  class="am-grid"
>
  <div
    class="am-flexbox am-flexbox-justify-center am-flexbox-align-stretch"
  >
    <div
      class="am-flexbox-item am-grid-item"
      style="width:20%;"
    >
      <div
        class="am-grid-item-content"
      >
        <div
          class="am-grid-item-inner-content column-num-5"
        >
          <svg
            class="am-icon am-icon-search am-icon-xxs"
          >
            <use
              xlink:href="#search"
            />
          </svg>
          <div
            class="am-grid-text"
          >
            xxs
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-flexbox-item am-grid-item"
      style="width:20%;"
    >
      <div
        class="am-grid-item-content"
      >
        <div
          class="am-grid-item-inner-content column-num-5"
        >
          <svg
            class="am-icon am-icon-search am-icon-xs"
          >
            <use
              xlink:href="#search"
            />
          </svg>
          <div
            class="am-grid-text"
          >
            xs
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-flexbox-item am-grid-item"
      style="width:20%;"
    >
      <div
        class="am-grid-item-content"
      >
        <div
          class="am-grid-item-inner-content column-num-5"
        >
          <svg
            class="am-icon am-icon-search am-icon-sm"
          >
            <use
              xlink:href="#search"
            />
          </svg>
          <div
            class="am-grid-text"
          >
            sm
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-flexbox-item am-grid-item"
      style="width:20%;"
    >
      <div
        class="am-grid-item-content"
      >
        <div
          class="am-grid-item-inner-content column-num-5"
        >
          <svg
            class="am-icon am-icon-search am-icon-md"
          >
            <use
              xlink:href="#search"
            />
          </svg>
          <div
            class="am-grid-text"
          >
            md
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-flexbox-item am-grid-item"
      style="width:20%;"
    >
      <div
        class="am-grid-item-content"
      >
        <div
          class="am-grid-item-inner-content column-num-5"
        >
          <svg
            class="am-icon am-icon-search am-icon-lg"
          >
            <use
              xlink:href="#search"
            />
          </svg>
          <div
            class="am-grid-text"
          >
            lg
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
